modules/imgcodecs/src/grfmt_exr.cpp

```diff
@@ -691,6 +691,16 @@ bool  ExrEncoder::write( const Mat& img, const std::vector<int>& params )
                 CV_Error(Error::StsBadArg, "IMWRITE_EXR_COMPRESSION is invalid or not supported");
             }
         }
+        if (params[i] == IMWRITE_EXR_DWA_COMPRESSION_LEVEL)
+        {
+#if defined(OPENEXR_VERSION_MAJOR) && OPENEXR_VERSION_MAJOR >= 3
+            header.dwaCompressionLevel() = params[i + 1];
+#elif defined(OPENEXR_VERSION_MAJOR)
+            throw std::runtime_error("Setting `IMWRITE_EXR_DWA_COMPRESSION_LEVEL` not supported in OpenEXR version " + std::to_string(OPENEXR_VERSION_MAJOR) + " (version 3 is required)");
```

Comment:
> `throw std::runtime_error`

Lets use `CV_LOG_ONCE_WARNING(NULL, "message");` instead to keep existed behavior for unknown/unsupported hints.

Reply:
Done